### DISCUSSION TITLE
[Bug]: [Deprecation] -ms-high-contrast-adjust is in the process of being deprecated.

### DISCUSSION THREAD

**Discussion by askpatrickw** [blakeblackshear/frigate#11898] [Report a Bug]
### Describe the problem you are having

[Deprecation] -ms-high-contrast-adjust is in the process of being deprecated. Please see https://blogs.windows.com/msedgedev/2020/09/17/styling-for-windows-high-contrast-with-new-standards-for-forced-colors/ for tips on updating to the new Forced Colors Mode standard._

### Steps to reproduce

Open the Config page and look in the console.

![image](https://github.com/blakeblackshear/frigate/assets/4002194/ec138e2d-44e9-4945-8ed0-2bf6c3c93f3f)

### Version

latest

### Frigate config file

```yaml
N\A
```

### Relevant log output

```shell
N\A
```

### Operating system

HassOS

### Install method

HassOS Addon

### Network connection

Wired

### Camera make and model

Revodata

**Comment by hawkeye217** [comment]
Thanks for your report, however this is not a bug, and the console messages are only a warning message about future deprecation of a CSS attribute used by the config editor package we use in Frigate. 

If you're experiencing problems and the editor doesn't actually function correctly, feel free to reopen this.
